hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/QuotaRepairTask.java

```diff
@@ -399,22 +399,18 @@ private static <VALUE> void extractCount(
       Table.KeyValue<String, VALUE> kv,
       Map<String, CountPair> prefixUsageMap,
       boolean haveValue) {
-    try {
-      String prefix = getVolumeBucketPrefix(kv.getKey());
-      CountPair usage = prefixUsageMap.get(prefix);
-      if (null == usage) {
-        return;
-      }
-      usage.incrNamespace(1L);
-      // avoid decode of value
-      if (haveValue) {
-        VALUE value = kv.getValue();
-        if (value instanceof OmKeyInfo) {
-          usage.incrSpace(((OmKeyInfo) value).getReplicatedSize());
-        }
+    String prefix = getVolumeBucketPrefix(kv.getKey());
+    CountPair usage = prefixUsageMap.get(prefix);
+    if (null == usage) {
+      return;
+    }
+    usage.incrNamespace(1L);
+    // avoid decode of value
```

Comment:
I guess the comment no longer applies, since `KeyValue` stores `value` already decoded.